### PR TITLE
chore(simulator): added react-flow and updated toolbar

### PR DESCRIPTION
**Pull request by SumantxD**
This PR aims to solve issue #179 

- added react flow library and build a basic layout
- updated the title bar for the Async option

solves: #179

### CODE REVIEW

**Comment by SumantxD** [comment]
file:///home/sumant/Pictures/Screenshots/Screenshot%20from%202023-07-10%2018-04-39.png

**Comment by SumantxD** [comment]
Screenshot from 2023-07-10 18-06-03

**Comment by NektariosFifes** [comment]
@SumantxD I will try to finish reviewing it today tomorrow and write my comments.

File: Desktop/src/main/preload.js [hunk 1]
```diff
@@ -1,4 +1,5 @@
 const { contextBridge, ipcRenderer } = require('electron');
+const os = require('os')
```
Comment (NektariosFifes) [inline Desktop/src/main/preload.js:3]: os does not need to be imported

**Review by NektariosFifes — COMMENTED** [review]
I will continue the review and see to the visual issues present. These are the easily fixable issues.

File: Desktop/src/renderer/App.tsx
Comment (NektariosFifes) [inline Desktop/src/renderer/App.tsx:1]: I think Editor should not be removed the flow component should be placed in the scenario workbench component. The point of the Editor container/component is to encapsulate information that components can use and share with each other using context.

**Comment by NektariosFifes** [comment]
The application is not the same as in the picture provided. Maybe you are missing some files or have not pushed a commit?

File: Desktop/src/renderer/App.tsx
Comment (SumantxD) [reply Desktop/src/renderer/App.tsx:1]: got it, will fix that

File: Desktop/src/main/preload.js [hunk 2]
```diff
@@ -1,4 +1,5 @@
...
```
[the remark this answers is not in the document]
Comment (SumantxD) [reply Desktop/src/main/preload.js]: will fix that :+1:

**Comment by SumantxD** [comment]
> @SumantxD The application is not the same as in the picture provided. Maybe you are missing some files or have not pushed a commit?

Fixed .

**Review by NektariosFifes — APPROVED** [review]

**Pull request merged**